### DISCUSSION TITLE
The heat release rate gradually decreases after removing the fire source

### DISCUSSION THREAD

**Discussion by uihiuy** [firemodels/fds#13172] [Q&A]
I built a pmma combustion model. The thickness of pmma is 5mm, and the ignition time is set to 25s. I tried to change the ignition time to 55s, but the heat release rate after removing the fire source is still small, close to 0. How to set the normal combustion of pmma [
[侧向火.txt](https://github.com/user-attachments/files/16229401/default.txt)
](url)

**Comment by drjfloyd** [comment]
You are predicting the burning rate which means you are doing research. This is not a solved problem.  A couple observations on your case:
-Your domain height is very short at 4 cm. Most of the flame height is outside your domain which is going to reduce the potential radiative flux to the PMMA.
-Your ignition source is not very physical. Your 2500 C fixed wall temperature is exposing the PMMA grid cells right near the source to 600 kW/m2.  This is significantly higher than the flux one would expect from a line burner ignition source or typical radiant panel.

**Reply by uihiuy** [reply]
So how much height should I be set appropriately? And how many ℃ should be set for the current ignition source .Thanks

**Reply by uihiuy** [reply]
I tried to change the duration of the gun but the heat release rate continued to drop close to 0, that's why

**Reply by drjfloyd** [reply]
You should see what the impact of height is on the results. I can't make the decision of an appropriate domain size for you, this is not my research. Same with the ignition source. What type of real world source are you attempting to represent? What is the size of that source and what heat flux would it provide?

**Reply by uihiuy** [reply]
Thank you but I still have a problem is that after I remove the fire source, the heat release rate and the mass loss rate are close to 0. What is this going on

**Reply by drjfloyd** [reply]
If you are tryring to predict the heat release rate, you are doing research. You need to think about what outputs can you add to  understand what is going on and investigate those outputs to determine what is happening.
